blocks/api/raw-handling/special-comment-converter.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Browser dependencies
+ */
+const { COMMENT_NODE } = window.Node;
+
+/**
+ * Looks for `<!--more-->` comments, as well as the `<!--more Some text-->`
+ * variant and its `<!--noteaser-->` companion, and replaces them with a custom
+ * element representing a future block.
+ *
+ * The custom element is a way to bypass the rest of the `raw-handling`
+ * transforms, which would eliminate other kinds of node with which to carry
+ * `<!--more-->`'s data: nodes with `data` attributes, empty paragraphs, etc.
+ *
+ * The custom element is then expected to be recognized by any registered
+ * block's `raw` transform.
+ *
+ * @param {Node} node The node to be processed.
+ * @return {void}
+ */
+export default function( node ) {
```

Comment:
Do you think we should unit test this or is it already covered elsewhere?

Reply:
I think some tests are in order 👍